src/vivarium_testing_utils/fuzzy_checker.py

```diff
@@ -213,6 +214,9 @@ def _calculate_bayes_factor(
         try:
             return bug_marginal_likelihood / no_bug_marginal_likelihood
         except (ZeroDivisionError, FloatingPointError):
+            logger.warning(
```

Comment:
Apparently if you hit these errors in the except block here they do not show up in the caplog fixture.

Reply:
Isn't that what we want? I don't see why we should show the user a warning here.

Reply:
I need to log it to test it. Perhaps I can do a different log level.

Reply:
It seems odd to log something only for testing purposes. Why not just test the return value?

Reply:
I'm also doing that and that is probably adequate enough.